pom.xml

```diff
@@ -1,66 +1,62 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.liquigraph.examples</groupId>
-  <artifactId>liquigraph-spring</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <packaging>jar</packaging>
-
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>1.4.2.RELEASE</version>
+    <version>1.4.3.RELEASE</version>
   </parent>
 
+  <groupId>org.liquigraph.examples</groupId>
+  <artifactId>liquigraph-spring</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
   <name>Example integration with Spring Boot</name>
 
   <properties>
-      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
I found it. In https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#encoding the value can also be used by the user property encoding. Therefore if only compiling is relevant using `encoding` is enough. Unfortunately no other plugin uses this convention. I learned something new there and the maven team is discussing a global single property for encoding (see https://cwiki.apache.org/confluence/display/MAVEN/POM+Element+for+Source+File+Encoding).